Dockerfile

```diff
@@ -61,5 +61,10 @@ RUN ln -s /usr/lib/x86_64-linux-gnu/libnvidia-ml.so.1 \
 COPY --from=build /opt/tabby /opt/tabby
 
 ENV TABBY_ROOT=/data
+ARG WRITE_DIRS="${TABBY_ROOT} ${TABBY_ROOT}/events"
```

Comment:
This won't work - as $TABBY_ROOT could be mounted to external directory, which might not writable to 1001 uid.

Reply:
Yes, this was the intended effect. To me it makes sense if the container don't write to the user file system as root, even if its mounted. 

Any alternate takes on how to solve this? Do you agree?

Reply:
The tradeoff being considered here is that we want to avoid inconveniencing the first-time user with the manual execution of `chmod` (as they are likely running `docker -it` directly).

One solution I have implemented is to create a new Docker image based on Tabby's distribution, similar to the one provided for Hugging Face deployment: https://huggingface.co/spaces/TabbyML/tabby-template-space/blob/main/Dockerfile

Reply:
The idea isn't that the user should run chmod, it's that the tabby container shouldn't modify user space, especially not as root. 

I agree that it should be as smooth as possible for new users. 

Where does tabby need to write today?